### PR TITLE
5448: Add cookie banner placeholders on videos

### PR DESCRIPTION
**Pull request by xendk**
Tell the user they have to accept cookies to view the video,

#### Link to issue

https://platform.dandigbib.org/issues/5448

#### Description

Add cookie banner placeholders on videos that tell the user they have to accept cookies to view the video.

#### Screenshot of the result

image

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

**Comment by xendk** [comment]
As usual, the behat test failures are unrelated.

File: modules/ding_content/templates/media-vimeo-video--cookieinformation.tpl.php [hunk 1]
```diff
@@ -20,5 +20,9 @@
  */
 ?>
 <div class="<?php print $classes; ?> media-vimeo-<?php print $id; ?>">
+  <div class="consent-placeholder" data-category="cookie_cat_statistic">
+    <p>Denne video er ikke tilgængelig da du ikke har accepteret marketing-cookies.</p>
```
Comment (cableman) [inline modules/ding_content/templates/media-vimeo-video--cookieinformation.tpl.php:24]: Should that have been wrapped in `t()`

File: modules/ding_content/templates/media-youtube-video--cookieinformation.tpl.php [hunk 2]
```diff
@@ -20,5 +20,9 @@
  */
 ?>
 <div class="<?php print $classes; ?> media-youtube-<?php print $id; ?>">
+  <div class="consent-placeholder" data-category="cookie_cat_statistic">
+    <p>Denne video er ikke tilgængelig da du ikke har accepteret marketing-cookies.</p>
```
Comment (cableman) [inline modules/ding_content/templates/media-youtube-video--cookieinformation.tpl.php:24]: Should that have been wrapped in `t()`

File: modules/ting_oembed/ting_oembed.module [hunk 3]
```diff
@@ -105,6 +105,18 @@ function ting_oembed_oembed_response_alter(&$response) {
       $iframe->setAttribute('src', '');
       $iframe->setAttribute('data-consent-src', $src);
       $iframe->setAttribute('data-category-consent', "cookie_cat_statistic");
+
+      $consent = $dom->createNode('div');
+      $consent->setAttribute('class', 'consent-placeholder');
+      $concent->setAttribute('data-category', "cookie_cat_statistic");
+
+      $concent->append($dom->createNode('p', 'Denne video er ikke tilgængelig da du ikke har accepteret marketing-cookies.'));
```
Comment (cableman) [inline modules/ting_oembed/ting_oembed.module:113]: Should that have been wrapped in `t()`

**Review by cableman — CHANGES_REQUESTED** [review]

File: modules/ding_content/templates/media-vimeo-video--cookieinformation.tpl.php (hunk 1 above)
Comment (xendk) [reply modules/ding_content/templates/media-vimeo-video--cookieinformation.tpl.php]: @cableman Fixed.

**Review by cableman — APPROVED** [review]

**Pull request merged**